zerver/test_email_mirror.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from django.test import TestCase
+
+from zerver.lib.test_helpers import (
+    AuthedTestCase, 
+    get_user_messages,
+    message_ids, message_stream_count,
+    most_recent_message,
+    queries_captured,
```

Comment:
A few of these imports are unneeded; e.g. queries_captured, message_ids, and message_stream_count; might as well clean these up to only be the stuff we actually need for these tests since we're making a new file...